tests/integration/testsuites/v2alpha1/manifests/certificate-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: api-gateway-webhook-certificate
+  namespace: kyma-system
+data:
+  tls.crt: ""
+  tls.key: ""
```

Comment:
arę these gonna be generated when applied on a cluster? Maybe I'd be better to inject some not-so-secure values here statically?

Reply:
Good catch, I can remove them, since I removed the tests. 😬